types/Support/Cache.php

```diff
@@ -0,0 +1,28 @@
+<?php
```

Comment:
You will need to add tests for the cases when a closure is given for the $default arguments.